docs/development/_index.md

```diff
@@ -7,10 +7,20 @@ This is a quickstart guide for building and running Headlamp for development.
 Please make sure you read the [Contribution Guidelines](../contributing.md) as well
 before starting to contribute to the project.
 
+See [platforms](../platforms.md) to find out which browsers, OS and flavours of Kubernetes we support.
+
+## Dependencies to get started
+
+These are the required dependencies to get started. Other dependencies are pulled in by the golang or node package managers (see frontend/package.json, app/package.json, backend/go.mod and Dockerfile).
+
+- [Node.js](https://nodejs.org/en/download/) Latest LTS (18.14 at time of writing). Many of us use [nvm](https://github.com/nvm-sh/nvm) for installing multiple versions of Node.
+- [Go](https://go.dev/doc/install), (1.19 at time of writing)
+- Make (GNU). On Windows this can be installed with the "chocolatey" package manager that is installed with node.
```

Comment:
should we add a link to make website too https://www.gnu.org/software/make/ ?? WDYT??

Reply:
Good idea.